.circleci/config.yml

```diff
@@ -0,0 +1,20 @@
+version: 2
+jobs:
+  build:
+    machine: true
```

Comment:
With the existing `Makefile`, I can easily deploy this change with `machine: true`. Otherwise, I need rewrite the whole codes, which I don't have the time now. 

My suggestion for you is, accept this PR first, that we have this validate feature. You or someone else can continue to work it out with docker way by Circle CI.